homeassistant/components/light/lifx.py

```diff
@@ -138,6 +143,28 @@ def aiolifx_effects():
                                config,
                                async_add_entities,
                                discovery_info=None):
+    """Set up the LIFX light platform.
+
+    Deprecated.
+    """
+    _LOGGER.warning('Loading LIFX via light platform config is deprecated.')
+    await _async_setup_platform(
```

Comment:
Since lifx is a dependency, configuring via a platform will set up the lifx component and load the config entry. If we also call this, we end up setting it up twice?

Reply:
I removed the backwards compatibility now.